pkg/controllers/wireguard_controller.go

```diff
@@ -460,8 +460,6 @@ func (r *WireguardReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			return ctrl.Result{}, err
 		}
 
-		bytes.Equal(b, secret.Data["state.json"])
```

Comment:
Any idea what the intention of this was?

Reply:
haha probably a mistake :D